RELEASE_MANAGEMENT.md

```diff
@@ -125,7 +125,12 @@ this are based on the Kubernetes process:
 * Extensions can be granted on a per-GEP basis
   * The owners of the GEP have to ask and provide a timeline (measured in
     days) as to when they believe the GEP will be ready for merge.
-* The request and approval for a GEP extension needs to be in public.
+* The request and approval for a GEP extension must be on the original GEP issue.
+* An extension can only be granted for a time period up to the distance from
+  the deadline when the extension is requested. So, an extension of a week
+  must be asked for at least a week out from a deadline, while an extension of
+  day must be asked for at least a day out from the deadline. Extension requests
+  not meeting this timing criteria will be denied.
```

Comment:
I really like this idea, but how do we handle the case where we get some significant blocking feedback on the last day before a deadline

Reply:
The only thing I can think of is extending the deadline extension policy to feedback - something like "we won't accept feedback unless you give the change owner a reasonable amount of time to make the changes before the deadline". But even then, you can't write a simple rule like this one, it will require some judgement.

tbh, I think we're better to handle this by reminding everyone to check outstanding GEP PRs and be willing to say "sorry, should have got in earlier" for _very_ late, blocking feedback.